app/src/main/java/com/thailam/mvp_sample/wallet/WalletActivity.java

```diff
@@ -0,0 +1,66 @@
+package com.thailam.mvp_sample.wallet;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+
+import com.thailam.mvp_sample.R;
+
+import java.util.List;
+
+public class WalletActivity extends AppCompatActivity implements WalletContract.View {
+
+	private WalletPresenter mWalletPresenter;
```

Comment:
em chỉnh lại code style
1 tab = 4 spaces nhé

Reply:
```suggestion
    private WalletContract.Presenter mPresenter;
```
Khai báo ntn thì contract ms có giá trị